obsolete/led-rgb-4pin-anode_rbg_v4.fzp

```diff
@@ -12,7 +12,7 @@
     </tags>
     <properties>
         <property name="package">5 mm [THT]</property>
-        <property name="family">LED</property>
+        <property name="family">//obsolete//LED</property>
```

Comment:
Which change required the obsoletion?

Reply:
Same as before:
The id of the part needs to finish in "LEDModuleID". The ending of the id triggers the code to create the object as an LED object, which is the one that has the code to change the color of the LEDs. As far as I know, changing the id forces to obsolete the parts or they will not be found.